packages/sdk-react/mui-styles.d.ts

```diff
@@ -41,6 +42,7 @@ declare module '@mui/material/styles' {
     MonitePayableStatusChip: Partial<PayableStatusChipProps>;
     MoniteApprovalRequestStatusChip: Partial<MoniteApprovalRequestStatusChipProps>;
     MoniteTablePagination: Partial<MoniteTablePaginationProps>;
+    MonitePayableDetailsInfo: Partial<PayableDetailsInfoProps>;
```

Comment:
I can't find a definition of the PayableDetailsInfoProps interface. Probably, forgot to add to the repo?